global.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Please remove the `global.json` file - I don't think it's helpful having it in the project.

Reply:
O_o it's so important.

Reply:
Not in this project it ain't.